CMakeLists.txt

```diff
@@ -98,21 +98,20 @@ set(DLT_USER "covesa"
 set(DLT_QNX_SLOG_ADAPTER_WAIT_BUFFER_TIMEOUT_MS "100"
     CACHE STRING "Timeout in milliseconds to wait before messages are dropped when input buffer is full")
 
-option(WITH_DLT_PKGCONFIG "Set to ON to generate pkgconfig .pc files"                                                ON)
-option(WITH_DLT_CXX11_EXT "Set to ON to build C++11 extensions"                                                      OFF)
-option(WITH_DLT_COREDUMPHANDLER "EXPERIMENTAL! Set to ON to build src/core_dump_handler binaries. EXPERIMENTAL"      OFF)
-option(WITH_DLT_LOGSTORAGE_CTRL_UDEV "PROTOTYPE! Set to ON to build logstorage control with udev support"            OFF)
-option(WITH_DLT_LOGSTORAGE_GZIP "Set to ON to build logstorage control with gzip compression support"                OFF)
-option(WITH_DLT_USE_IPv6 "Set to ON for IPv6 support"                                                                ON)
-option(WITH_DLT_KPI "Set to ON to build src/kpi binaries"                                                            OFF)
-option(WITH_DLT_FATAL_LOG_TRAP "Set to ON to enable DLT_LOG_FATAL trap(trigger segv inside dlt-user library)"        OFF)
-option(WITH_UDP_CONNECTION "Set to ON to enable dlt UDP multicast SUPPORT"                                           OFF)
-option(WITH_LIB_SHORT_VERSION "Set to ON to build library with only major number in version"                         OFF)
-option(WITH_DLT_DISABLE_MACRO "Set to ON to build code without Macro interface support"                              OFF)
-option(WITH_LEGACY_INCLUDE_PATH "Set to ON to add <prefix>/dlt to include paths
-                                 for the CMake config file, in addition to only <prefix>"                            ON)
-option(WITH_EXTENDED_FILTERING "Set to OFF to build without extended filtering.
-                                Json filter only supports Linux based system with json-c and QNX"                    OFF)
+option(WITH_DLT_PKGCONFIG     "Set to ON to generate pkgconfig .pc files"                                        ON)
```

Comment:
Please do not rework my alignment here, just add new option following the format.
Thanks